frontend/src/components/EditableList.vue

```diff
@@ -167,7 +227,15 @@ watch(
         class="group flex gap-8 border-b-1 border-blue-300 py-16"
         :class="{ 'border-t-1': index == 0 }"
       >
-        <component :is="summaryComponent" :data="entry" />
+        <component
+          :is="summaryComponent"
+          :ref="
+            (el: HTMLElement | ComponentPublicInstance | null) =>
+              el && editedItemRef.set(index, el)
+          "
```

Comment:
Cool, das kannte ich auch nicht. 🤩  

out-of-scope: Hier habe ich noch das Gefühl, dass es vielleicht schöner sein könnte, wenn das eine Component wäre und damit die ganze Behandlung mit den Indices wegfiele, weil es dann immer genau eine edit und eine display component gibt, die man einfach als ref zum parent emitten könnte, wenn der button geklickt wird. Wenn wir die EditableList irgendwann mal refactoren, könnte man das mal versuchen.